containers/zammad/Dockerfile

```diff
@@ -37,7 +37,7 @@ LABEL org.label-schema.build-date="$BUILD_DATE" \
       org.label-schema.vcs-url="https://github.com/zammad/zammad" \
       org.label-schema.vcs-type="Git" \
       org.label-schema.vendor="Zammad" \
-    org.label-schema.schema-version="5.1.0" \
+      org.label-schema.schema-version="5.1.0" \
```

Comment:
Maybe we can put the whole change in: https://github.com/zammad/zammad-docker-compose/pull/274 and adjust to 5.2 there?

Reply:
I could do that if needed, but actually this PR here can already be merged for stable. Node.js 16 works fine there, too. Then we have less changes later for 5.2. Let me know.